### PR TITLE
add a man page

### PR DESCRIPTION
**Pull request by retokromer**
- general functions and parameters are considered
- except the `serachlto` function, which is not completely clear to me
- CUNY TV's specific functions are not considered so far

If OK, I’ll merge and make a new release.

### CODE REVIEW

**Comment by retokromer** [comment]
Update: `searchlto` also uses CUNY TV's own structure.

File: ltopers.1 [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+.B -t
+cartridge's \fIserial
+.TP
+.B -e
+eject the cartridge after migration: \fIY\fR (default) or \fIN
+.TP
+.B -h
+display a help message
+.SS writelto -t \fIserial\fB [-e \fIY\fR|\fIN\fB] [-v] | -h
+Write the contents of a specified directory onto a mounted LTFS-formated LTO cartridge.
```
Comment (kieranjol) [inline ltopers.1]: should be LTFS-formatted

Comment (retokromer) [inline ltopers.1]: Thank you! I corrected this in the markdown version, where it was also wrong, but forgot to do so here…

**Comment by kieranjol** [comment]
manpage LGTM in terms of formatting and typos etc anyhow.

**Comment by privatezero** [comment]
Looks good to me!
Regarding searchlto, it isn't CUNY specific per-se, but was designed to facilitate the way that CUNY needs to search records across the LTO database.

**Comment by retokromer** [comment]
Thank you! Is there any available information about the structure of the database? That would be useful when one wishes to generalise.

**Comment by privatezero** [comment]
Currently there is a doc in progress about the database functionality - it is incomplete and needs revision (will be working on this soon). It does however have tables representing the structure!

It lives (for now) here: https://github.com/mediamicroservices/mm/blob/master/Resources/dbinfo.md

I don't think anyone external to CUNY has tested this, so I am sure there are unanticipated bugs, but in theory this could be setup on any computer running MySQL with the command `createpremisdb -c` and then configuring in `mmconfig`.

**Comment by privatezero** [comment]
I also am seriously considering renaming that script, as the DB has expanded far beyond gathering PREMIS data to also collect LTO schemas, fixity and perceptual hashes.

**Comment by retokromer** [comment]
👍 for renaming

**Comment by retokromer** [comment]
What about divide the commands in two groups:

- `brew install ltopers`
- `brew install ltopers --with-db`

I guess, this way it would be simpler for general users.

**Comment by retokromer** [comment]
My full proposal is at https://github.com/amiaopensource/homebrew-amiaos/issues/46.

**Pull request merged**